docs/courses/csintro1/loops/projectiles.md

```diff
@@ -281,7 +284,7 @@ projectile = sprites.createProjectile(img`
 1. Create the example code above 
 2. Modify the code to make the sprite start from the upper right corner by default and move toward the lower left corner 
 3. Place the code in a loop to repeat 4 times with a short pause between loop iterations
-4. **Challenge:** make sprites originate from each of the 4 corners inside the loop
+4. **Challenge:** Inside the loop, make sprites originate from each of the 4 corners of the screen.
```

Comment:
I've been doing lower case after colons throughout (i.e. 'inside' instead of 'Inside' here); I'm fine with changing if that's preferred but would want to stay consistent for all.

Same with trailing period on lists in tasks / examples~

Reply:
Yea, was thinking about the case of the period...if they sound like complete sentences, well maybe they need a period. Otherwise, short statements could go unterminated. However, a mix of the two in a list would look bad... pick your style

Reply:
I think I prefer lower case after colon and no period at the end as it currently is; most of the list items are ideally fairly short statements and it looks cleaner to me that way